spec/models/post_spec.rb

```diff
@@ -1,5 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'Validation for posts' do
+    user = User.create(name: 'paka', bio: 'Test user')
+    subject do
+      Post.new(title: 'paka post', text: 'This is text post', author_id: user)
+    end
+
+    before { subject.save }
+
+    it 'Is counter a number' do
+      subject.comments_counter = '1'
+      expect(subject).to_not be_valid
+    end
```

Comment:
- [ ] For the post spec, Kindly add validation tests for the following:
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than zero.
- LikesCounter must be an integer greater than zero.